spec/api-browser-window-spec.js

```diff
@@ -3250,15 +3251,13 @@ describe('BrowserWindow module', () => {
     }
 
     beforeEach(() => {
-      if (w != null) w.destroy()
-      w = new BrowserWindow({
+      iw = new BrowserWindow({
         show: false,
         webPreferences: {
           contextIsolation: true,
           preload: path.join(fixtures, 'api', 'isolated-preload.js')
         }
       })
-      if (ws != null) ws.destroy()
```

Comment:
It is better to keep this.
The `afterEach` hook won't be called if a test fails, and a window will be leaked.